assets/queries/cloudFormation/rds_public_interface/query.rego

```diff
@@ -0,0 +1,29 @@
+package Cx
+
+CxPolicy [ result ] {
+	resource := input.document[i].Resources[name]
+  resource.Type == "AWS::RDS::DBInstance"
+  object.get(resource.Properties, "PubliclyAccessible", "undefined") == "undefined"
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("Resources.%s.Properties",  [name]),
+                "issueType":		"MissingAttribute", 
+                "keyExpectedValue": sprintf("'Resources.%s.Properties.PubliclyAccessible' is defined", [name]),
+                "keyActualValue": 	sprintf("'Resources.%s.Properties.PubliclyAccessible' is undefined", [name]),
+              }
+}
```

Comment:
> The default behavior value depends on your VPC setup and the database subnet group. For more information, see the PubliclyAccessible parameter in CreateDBInstance in the Amazon RDS API Reference.

Since the default value is variable, we should not check if the attribute is missing and assume that it's public available. We can cut this part of the query

Reply:
Understood. Thanks.